ws/src/main/java/org/kaazing/netx/ws/internal/io/OutgoingSentinelExtension.java

```diff
@@ -54,10 +54,15 @@
 
     private final ClosePayloadRO closePayloadRO;
     private final ByteBuffer closePayload;
+    private final byte[] frameBuffer;
+
+    private int frameBufferOffset;
```

Comment:
Try to remove this member variable if possible.  Seems like we might only need call stack references to this state now.